dwave_networkx/algorithms/partition.py

```diff
@@ -0,0 +1,157 @@
+# Copyright 2021 D-Wave Systems Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+from dimod import ConstrainedQuadraticModel
+from dimod import quicksum 
+from dimod import Binary
+import numpy as np

```

Comment:
It appears that you never use this import, which is good because otherwise we would need to add a dependency on numpy to the requirements.